### PR TITLE
Add gfx-rs news to newsletter 9

### PR DESCRIPTION
**Pull request by kvark**
_Part of #111 ("Newsletter 9: Coordination/Tracking")_

### CODE REVIEW

File: content/posts/newsletter-009/index.md [hunk 1]
```diff
@@ -54,6 +54,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### gfx-rs and wgpu news
+
```
Comment (ozkriff) [inline content/posts/newsletter-009/index.md:58]: Maybe there's some image from the updates that can be used here? Any ideas?

**Comment by kvark** [comment]
Does this one fit? This is "Hectic" game prototype, running on wgpu-rs (including the browsers!).
Source - https://github.com/expenses/hectic-rs

wgpu-app-1588257877

**Comment by ozkriff** [comment]
yeah, looks cool :)

**Comment by expenses** [comment]
Here's another screenshot in case you were in need of one! screenshot

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**